src/internal/kopia/filesystem.go

```diff
@@ -0,0 +1,35 @@
+package kopia
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/kopia/kopia/repo/blob/filesystem"
+
+	"github.com/alcionai/corso/src/pkg/control/repository"
+	"github.com/alcionai/corso/src/pkg/storage"
+)
+
+func filesystemStorage(
+	ctx context.Context,
+	repoOpts repository.Options,
+	s storage.Storage,
+) (blob.Storage, error) {
+	cfg, err := s.StorageConfig()
+	if err != nil {
+		return nil, clues.Stack(err).WithClues(ctx)
+	}
+
+	fsCfg := cfg.(*storage.FilesystemConfig)
+	opts := filesystem.Options{
+		Path: fsCfg.Path,
+	}
+
+	store, err := filesystem.New(ctx, &opts, false)
```

Comment:
The final parameter seems to be whether kopia should attempt to create the parent directories for the repo. I get that always setting it to false makes tests easier, but can it lead to failures during CLI usage? We don't seem to have code at the moment to handle intermediate directory creation

This can be addressed later, but at the very least we should add a note to the CLI help explaining what directories need to exist prior to running corso so that users aren't surprised